src/oop/programs_with_multiple_classes/hard/q2/QuestionText.md

```diff
@@ -0,0 +1,39 @@
+# oop-programs_with_multiple_classes-hard-q2
+
+You want to write a program that models a group of people and their blanket fort. To do this, create a Java program with
+three classes: `Person`, `Blanket`, and `TestFort`. See the requirements below for each class. Note that a `toString`
+method has been provided in the `Person` and `Blanket` classes for you to use.
+
+## Person Class
+
+Your `Person` class should have the following items:
+
+- Attributes:
+    - A `double` called **height**
```

Comment:
We should maybe also say something about it not being accessible from another class

Reply:
You did not address this comment

Reply:
They need to know if the attribute should be public vs private